common/djangoapps/third_party_auth/models.py

```diff
@@ -946,3 +948,5 @@ class Meta:
         app_label = "third_party_auth"
         verbose_name = "Provider Configuration (LTI)"
         verbose_name_plural = verbose_name
+
+register(UserSocialAuth, app=__package__)
```

Comment:
Please add some docstring/comment as to why this call is done here and what will be its impact

Reply:
Agreed that a comment here would be good.

Reply:
Specifically, it's probably worth mentioning here that this makes `third_party_auth` a required installed-app for Studio, even though it's only meant to be used in LMS.

I also feel like this should be near the top of the file instead of the bottom, just because it would be easier to see there, but I don't feel strongly about this. Just a suggestion.